### DISCUSSION TITLE
通过ide插件市场安装 Go Linter 成功之后，打开ide 工具选项栏 配置 golangci-lint 在Path to golangci-lint:选项框中 点击Open按钮选中golangci-lint 文件，发现一直打开不了，能帮忙看看嘛？ 系统 win10 ;ide版本GoLand 2022.3.2 ； GO linter 插件版本 1.6.3

### DISCUSSION THREAD

**Discussion by flechazo-x** [xxpxxxxp/intellij-plugin-golangci-lint#134] [Q&A]
![image](https://user-images.githubusercontent.com/70946742/218080452-ddb2075a-41eb-43d3-ac70-34c33ac84314.png)
![image](https://user-images.githubusercontent.com/70946742/218080678-d783e536-e179-4e61-afb0-5fcc21bd388b.png)

**Comment by xxpxxxxp** [comment]
结合图中的图标来看，这可能不是一个可执行文件，或者说当前用户没有执行的权限，可以试着在非管理员状态的  CMD 里运行一下看看
